holoviews/plotting/bokeh/element.py

```diff
@@ -1393,20 +1372,23 @@ def _setup_autorange(self):
               }}
             }}
 
-            if (y_range_name) {{
+            if (y_range_name in range_limits) {{
+              const [vmin_old, vmax_old] = range_limits[y_range_name]
+              range_limits[y_range_name] = [Math.min(vmin, vmin_old), Math.max(vmax, vmax_old)]
```

Comment:
This is the core of the change, the rest of the PR is just cleanup.

Reply:
So `range_limits` was accumulating a bunch of mins and maxes that were not being used!

One thing though, shouldn't it segregate aggregations separately on the left/right axis if you have twin axes enabled?

Reply:
That's what the y_range_name is for.

Reply:
Ok. I see that `y_range_name = renderer.model.y_range_name` and now I remember that this is how Bokeh defined multiple axes. Thanks for explaining!